fs/torrentfs.go

```diff
@@ -60,14 +60,14 @@ var (
 )
 
 func isSubPath(parent, child string) bool {
-	if len(parent) == 0 {
```

Comment:
@withshubh I need some justification for this.

Reply:
@anacrolix Please have a look at this issue description [here](https://deepsource.io/gh/withshubh/torrent/issue/CRT-A0004/description/).

Reply:
That seems fair.